### PR TITLE
add regnet chinese doc

### PR DESCRIPTION
**Pull request by a-strong-python**
Add Chinese reference documents to the regnet model

### CODE REVIEW

**Comment by a-strong-python** [comment]
@vanpelt

**Comment by ArthurZucker** [comment]
cc @stevhliu

File: docs/source/zh/model_doc/regnet.md [hunk 1]
```diff
@@ -0,0 +1,498 @@
+# RegNet
+
+## 概述
+
+RegNet模型是由Ilija Radosavovic，Raj Prateek Kosaraju，Ross Girshick，Kaim He，Piotr Dollár在[《设计网络设计空间](https://arxiv.org/abs/2003.13678)》中提出的。
```
Comment (stevhliu) [inline docs/source/zh/model_doc/regnet.md:5]:
```suggestion
RegNet模型是由Ilija Radosavovic，Raj Prateek Kosaraju，Ross Girshick，Kaim He，Piotr Dollár在 [设计网络设计空间](https://arxiv.org/abs/2003.13678) 中提出的。

```

File: docs/source/zh/model_doc/regnet.md [hunk 2]
```diff
@@ -0,0 +1,498 @@
...
+
+来自[野外视觉特征的自监督预训练](https://arxiv.org/abs/2103.01988)的巨大 10B 模型， 在 10 亿张 Instagram 图像上进行训练，可在[中心](https://huggingface.co/facebook/regnet-y-10b-seer)获得
+
+
+
+## 资源
+
+官方 Hugging Face 和社区（由 🌎 ）资源列表，可帮助您开始使用 RegNet。
+
+图像分类
```
Comment (stevhliu) [inline docs/source/zh/model_doc/regnet.md:23]: Should use the special doc-builder syntax for this

```suggestion
<PipelineTag pipeline="image-classification"/>
```

**Review by stevhliu — COMMENTED** [review]
Thanks for translating this! 🤗 

Everything after the `## Resources` section can be left untranslated. These classes/methods are automatically documented with our special doc-builder syntax, [`[[autodoc]]`](https://github.com/huggingface/transformers/blob/4830f2696575988faee4af78b6049b62a750ecd4/docs/source/en/model_doc/regnet.md?plain=1#L47).

**Comment by a-strong-python** [comment]
> Thanks for translating this! 🤗 
> 
> Everything after the `## Resources` section can be left untranslated. These classes/methods are automatically documented with our special doc-builder syntax, [`[[autodoc]]`](https://github.com/huggingface/transformers/blob/4830f2696575988faee4af78b6049b62a750ecd4/docs/source/en/model_doc/regnet.md?plain=1#L47).

I know that the `## Resources` section can automatically generate documents, but in this case most of the documentation comments will still be in English and cannot be translated into Chinese.

**Pull request closed without merging**